### ISSUE TITLE
Cannot specify a log level from the CLI

### ISSUE THREAD

**Issue by NotMyself** [aspnet/dotnet-watch#142]
I am attempting to write an interactive tutorial and want to use dotnet-watch. I would like to suppress the info messages printed out by the watch command so that only messages from my tutorial are displayed.

I have tried adding a logging.json file, but it doesn't seem to be respected. The log level in the Tools project appears to be [hard coded to info](https://github.com/aspnet/dotnet-watch/blob/dev/src/Microsoft.DotNet.Watcher.Tools/CommandOutputProvider.cs#L27). How can I override it?

**Comment by victorhurdugaci** [comment]
Agree, it would be a nice feature. It's a bit tricky to implement because we need a way to pass arguments to `dotnet watch`. Today, it just passes everything to `dotnet`. We'll definitely investigate this

**Comment by NotMyself** [comment]
It might be as simple as looking for a logging.json so it can be configured externally.

**Comment by muratg** [comment]
Assigning to @victorhurdugaci for invesigation in 1.1.0.

**Comment by victorhurdugaci** [comment]
Hey @glennc, what do you think about this one?

**Comment by glennc** [comment]
hmmm. What are examples of the output that you want to suppress? Is it all output from watch?

I am thinking a switch passed to watch to stop it from logging anything. `-[q]uite`

**Comment by victorhurdugaci** [comment]
Talked to @glennc offline. We'll make this configurable through and environment variable, just like we do for enabling the polling watcher.

**Comment by NotMyself** [comment]
Here is an example from the [DotNetCoreKoans](https://github.com/NotMyself/DotNetCoreKoans) project. The project is a teaching tool where the student is instructed to open the project in a text editor and then run `dotnet watch run` in their console. The student then changes code in a set of unit tests learning about the C# language. The output of the application guides them  through each tutorial.

When running the application without the watcher, I have complete control over the output. With the watcher, it clutters up the output making it hard to follow. 

At a minimum it would be nice to have the watcher respect a config file or a flag to specify silent output.

![screen shot 2016-08-30 at 3 16 11 pm](https://cloud.githubusercontent.com/assets/73120/18109185/bae0c3da-6ec4-11e6-928b-bfc8e3af1a10.png)

**Comment by muratg** [comment]
Would we be still displaying errors in "quiet" mode, and just be silent otherwise? Or completely silent?

**Comment by victorhurdugaci** [comment]
The value you set in the env var will be the value from the LogLevel enum. If you set it to `none` nothing will be displayed while if you set it to `error` only error (and above) will be displayed. So, it's not a "quiet mode"

**Comment by victorhurdugaci** [comment]
Hey @NotMyself , I added a new flag to allow you to turn off the log messages. See the updated readme for usage.

**Comment by natemcmaster** [comment]
In #197, we added `--verbose` and `--quiet` options to dotnet-watch. 

We also removed the env variable added in #155. It's worth noting that #197 also added support for the [global dotnet-cli verbose flag](https://github.com/dotnet/cli/blob/v1.0.0-preview2/src/Microsoft.DotNet.Cli.Utils/CommandContext.cs#L13) (usage `dotnet --verbose watch`) which can also be set via the env variable `DOTNET_CLI_CONTEXT_VERBOSE=true`
